### ISSUE TITLE
View menu shortcuts not functioning.

### ISSUE THREAD

**Issue by Xaekai** [hjnilsson/rme#299]
ArchLinux, compiled build, no patches.
Built with:
```
cmake ../ \
-D CMAKE_BUILD_TYPE:STRING=RELEASE \
-D CMAKE_CXX_FLAGS:STRING="-march=native" \
-D OpenGL_GL_PREFERENCE:STRING=GLVND \
-D wxWidgets_CONFIG_EXECUTABLE:FILEPATH=/bin/wx-config-gtk3 

make -j $(nproc)
```
my local repo is current (afb7417a5562baabd2bd455b665ea22fc7335125)

The program runs fine but most the shortcuts in the View menu do not work. This may or may not be related to #287.

I went through and checked every single item in the menu. All of them do not work _except the zoom subsection_. The zooms/reset shortcuts work fine. Shortcuts from other menus also work.

**Comment by Xaekai** [comment]
I'm also gonna guess this may be related to #231 as well.

**Comment by Xaekai** [comment]
I've compiled a DEBUG build and on that binary in additional to the View menu shortcuts not working, the shortcuts for the About menu and File menu also do not work.

However, I was able to get all the shortcuts to work if I brought up the Terrain palette and had a dropdown box selected. Very strange. I will capture a video of this behavior since this bug seems to have been around such a long time and gone unfixed. I can only presume it was from a lack of sufficient report data to work off of.
